packages/coinstac-ui/app/render/components/computation-field-basic.js

```diff
@@ -64,13 +62,20 @@ export default class ComputationFieldBasic extends Component {
     if (type === 'select') {
       controlProps.componentClass = 'select';
       controlProps.multiple = true;
-      controlProps.value = Array.isArray(value) ? value : [];
       formGroupProps.className = 'computation-field-select';
 
+      /**
+       * Redux Form transforms single-item arrays into that singular value.
+       * Adjust it for the `select`.
+       */
+      if (!Array.isArray(value) && value !== '') {
+        controlProps.value = [value];
+      }
+
       formControl = (
         <FormControl {...controlProps}>
           {options.map((option, index) => {
-            return <option key={index} value={index}>{option}</option>;
+            return <option key={index} value={option}>{option}</option>;
```

Comment:
No longer using indices as values and performing the following array lookup: just use the RoI value!